libs/designer/src/lib/ui/panel/panelTabs/parametersTab/index.tsx

```diff
@@ -301,7 +301,9 @@ const ParameterSection = ({
             token.key === constants.FOREACH_CURRENT_ITEM_KEY
           );
         }
-        return supportedTypes.some((supportedType) => equals(supportedType, token.type));
+        return supportedTypes.some((supportedType) => {
+          return Array.isArray(token.type) ? false : equals(supportedType, token.type);
```

Comment:
nit: Simplify the expression as !Array.isArray(token.type) && equals(type, token.type)